src/test/java/redis/clients/jedis/commands/jedis/ControlCommandsTest.java

```diff
@@ -184,6 +184,13 @@ public void onCommand(String command) {
   public void configGet() {
     List<String> info = jedis.configGet("m*");
     assertNotNull(info);
+    info = jedis.configGet("*max-*-entries*", "maxmemory");
+    assertNotNull(info);
+    assertEquals(12, info.size());
```

Comment:
This is risky. Any future change in Redis may cause this test to fail.

Just test:
- There is at least one entry matching `*max-*-entries*` at even (0-based) position.
- There is `maxmemory` at even (0-based) position.

Or, simply:
- There is `maxmemory` at even (0-based) position.
- Size of the list is at least 4.